### PR TITLE
Add compat data for api.Window.find

### LINKED ISSUE #3464: window.find is missing data
https://developer.mozilla.org/en-US/docs/Web/API/Window/find#Browser_compatibility has no compatibility table.

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Fixes #3464.

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -509,6 +509,54 @@
...
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
```
Comment (stof) [inline api/Window.json]: according to the bugzilla ticket mentioned in the issue, this should be `false`, not `null`

Comment (vinyldarkscratch) [reply api/Window.json]: There's only an MDN link on the issue, and neither of the bugs on that page mention Edge or IE (at least, not that I can see).  I also wouldn't really count on bug reports for Firefox having fully accurate data for IE.

Can you link to the Bugzilla ticket you're referring to?

Comment (stof) [reply api/Window.json]: hmm, I was referring to https://bugzilla.mozilla.org/show_bug.cgi?id=672395
The link is in the MDM page indeed.

Comment (vinyldarkscratch) [reply api/Window.json]: Ah, no wonder I didn't find it the first time around -- they're using the abbreviation for IE (which I didn't look for).  Can now confirm that both IE and Edge do not support this.

File: api/Window.json [hunk 2]
```diff
@@ -509,6 +509,54 @@
           }
         }
       },
+      "find": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/find",
+          "support": {
+            "chrome": {
+              "version_added": true
```
Comment (jpmedley) [inline api/Window.json]: The [commit for this](https://chromium.googlesource.com/chromium/src/+/f813c3fae96d7b31f71d93595ff71363700952a1) predates version 1. Therefore `chrome` is `, `chrome_android` is 18, and `webview_android` is `true`.

**Comment by vinyldarkscratch** [comment]
Rebase complete!

**Pull request merged**